apps/foundation-demo/src/app/checkout/components/shipping/shipping-options/shipping-options.component.spec.ts

```diff
@@ -7,7 +7,7 @@ import { ShippingOption, ShippingFactory } from '@daffodil/core';
 
 let shippingFactory: ShippingFactory = new ShippingFactory();
 let stubShippingOptions = shippingFactory.createShippingOptions();
-let stubSelectedShippingOption = stubShippingOptions[0].id;
+let stubSelectedShippingOption = stubShippingOptions[0].text;
```

Comment:
This was originally changed because the shipping summary is now displaying the selectedShippingOption, which it wasn't before. After some thought, I decided it would be simpler and better to just make shippingOptions a list of strings instead of a list of ShippingOptions. Sorry for the big commits. I wasn't totally sure where I was going for a bit, but I think I arrived at a better way to handle shippingOptions. This also allows for the ShippingOption model and the ShippingFactory to be deleted, so that's nice too.